make-debs.sh

```diff
@@ -33,7 +33,7 @@ git clean -dxf
 # c) compares higher than any previous commit
 # d) contains the short hash of the commit
 #
-vers=$(git describe --match "v*" | sed s/^v//)
+vers=`echo 'stx-ceph_v13.2.0' | sed 's/^stx-ceph_v//'`
```

Comment:
Why bother with sed here if you're going to hard-code the response?

Reply:
@dtroyer : 
1) Currently, it's hard-code here.
2) It's better to use tag to get this info
    git describe --match "v*" | sed 's/^v//'
3) There's no tag in stx-ceph v13.2.0 branch.
I think we could give a tag later.

Reply:
@dtroyer : Do you agree with that we mark the tag as "stx-ceph_v13.2.0" in stx-ceph code base?